docs/reference/search/explain.asciidoc

```diff
@@ -150,12 +141,10 @@ This will yield the same result as the previous request.
     string. Defaults to the default search analyzer.
 
 `analyze_wildcard`::
-    Should wildcard and prefix queries be analyzed or
-    not. Defaults to false.
+    Should wildcard and prefix queries be analyzed or not. Defaults to false.
 
 `lenient`::
-    If set to true will cause format based failures (like
-    providing text to a numeric field) to be ignored. Defaults to false.
+    If set to true will cause format based failures (like providing text to a numeric field) to be ignored. Defaults to false.
 
```

Comment:
Similar to above. I'd probably go with, "Ignore format based failures such as providing text to a numeric field.